src/components/enterprise-user-subsidy/enterprise-offers/data/hooks.js

```diff
@@ -48,48 +41,21 @@ export const useEnterpriseOffers = ({
     }
   }, [enterpriseId, enableOffers]);
 
-  // Fetch enterprise coupons to determine if the enterprise offers can be used to enroll
   useEffect(() => {
-    const fetchEnterpriseCoupons = async () => {
-      try {
-        const response = await fetchCouponsOverview(
-          { enterpriseId },
-        );
-        const { results } = camelCaseObject(response.data);
-        setEnterpriseCoupons(results);
-      } catch (error) {
-        logError(error);
-      } finally {
-        setIsLoadingEnterpriseCoupons(false);
-      }
-    };
-
-    if (enableOffers) {
-      fetchEnterpriseCoupons();
-    } else {
-      setIsLoadingEnterpriseCoupons(false);
-    }
-  }, [enableOffers, enterpriseId]);
-
-  useEffect(() => {
-    if (!enableOffers || isLoading) {
+    if (!enableOffers || isLoadingOffers) {
       return;
     }
 
-    // For MVP we will only support enterprises with one active offer
-    const enterpriseHasOneOffer = enterpriseOffers.length === 1;
+    if (enterpriseOffers.length > 0) {
+      setCanEnrollWithEnterpriseOffers(true);
 
-    if (!enterpriseHasOneOffer) {
-      return;
+      const hasLowBalance = enterpriseOffers.some(offer => offer.isLowOnBalance);
+      const hasNoBalance = enterpriseOffers.every(offer => offer.isOutOfBalance);
```

Comment:
These two lines construct the confusion matrix depicted here (note: the current threshold for "out of balance" in code is $99, but confusion matrix lists $149; clarifying with UX/Product which to use moving forward): 

<img width="631" alt="image" src="https://user-images.githubusercontent.com/2828721/200379473-940b19bb-9c7a-47c6-9e4a-9978e0a45e3d.png">

Reply:
Confirmed we'll be sticking to the existing $99 threshold for "no balance"